### ISSUE TITLE
Docs don't build with sphinx 7.2

### ISSUE THREAD

**Issue by Mab879** [ComplianceAsCode/content#11011]
#### Description of problem:
Docs don't build with sphinx 7.2

#### SCAP Security Guide Version:
2ea9497d44fd969e3356aadec53aa44df9751418

#### Operating System Version:
Fedora 38

#### Steps to Reproduce:

1. build docs with sphinx 7.2+

#### Actual Results:
Build fails

#### Expected Results:
Build is successful 

#### Additional Information/Debugging Steps:
This issue appears to be a sphinx issue: https://github.com/sphinx-doc/sphinx/issues/11632

```
Traceback (most recent call last):
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/cmd/build.py", line 298, in build_main
    app.build(args.force_all, args.filenames)
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/application.py", line 355, in build
    self.builder.build_update()
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/builders/__init__.py", line 293, in build_update
    self.build(to_build,
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/builders/__init__.py", line 313, in build
    updated_docnames = set(self.read())
                           ^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/builders/__init__.py", line 420, in read
    self._read_serial(docnames)
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/builders/__init__.py", line 441, in _read_serial
    self.read_doc(docname)
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/builders/__init__.py", line 498, in read_doc
    publisher.publish()
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/core.py", line 217, in publish
    self.document = self.reader.read(self.source, self.parser,
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/io.py", line 105, in read
    self.parse()
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/readers/__init__.py", line 78, in parse
    self.parser.parse(self.input, document)
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/parsers.py", line 81, in parse
    self.statemachine.run(inputlines, document, inliner=self.inliner)
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 170, in run
    results = StateMachineWS.run(self, input_lines, input_offset,
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/statemachine.py", line 240, in run
    context, next_state, result = self.check_line(
                                  ^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/statemachine.py", line 452, in check_line
    return method(match, context, next_state)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 2779, in underline
    self.section(title, source, style, lineno - 1, messages)
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 327, in section
    self.new_subsection(title, lineno, messages)
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 393, in new_subsection
    newabsoffset = self.nested_parse(
                   ^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 281, in nested_parse
    state_machine.run(block, input_offset, memo=self.memo,
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 196, in run
    results = StateMachineWS.run(self, input_lines, input_offset)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/statemachine.py", line 240, in run
    context, next_state, result = self.check_line(
                                  ^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/statemachine.py", line 452, in check_line
    return method(match, context, next_state)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 2779, in underline
    self.section(title, source, style, lineno - 1, messages)
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 327, in section
    self.new_subsection(title, lineno, messages)
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 393, in new_subsection
    newabsoffset = self.nested_parse(
                   ^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 281, in nested_parse
    state_machine.run(block, input_offset, memo=self.memo,
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 196, in run
    results = StateMachineWS.run(self, input_lines, input_offset)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/statemachine.py", line 240, in run
    context, next_state, result = self.check_line(
                                  ^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/statemachine.py", line 452, in check_line
    return method(match, context, next_state)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 2352, in explicit_markup
    nodelist, blank_finish = self.explicit_construct(match)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 2364, in explicit_construct
    return method(self, expmatch)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 2101, in directive
    return self.run_directive(
           ^^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/docutils/parsers/rst/states.py", line 2151, in run_directive
    result = directive_instance.run()
             ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/ext/autodoc/directive.py", line 139, in run
    documenter.generate(more_content=self.content)
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/ext/autodoc/__init__.py", line 869, in generate
    if not self.parse_name():
           ^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/ext/autodoc/__init__.py", line 1001, in parse_name
    ret = super().parse_name()
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/ext/autodoc/__init__.py", line 388, in parse_name
    assert matched is not None
           ^^^^^^^^^^^^^^^^^^^
AssertionError

Exception occurred:
  File "/home/mburket/code/ComplianceAsCode/content/venv/lib64/python3.11/site-packages/sphinx/ext/autodoc/__init__.py", line 388, in parse_name
    assert matched is not None
           ^^^^^^^^^^^^^^^^^^^
AssertionError
The full traceback has been saved in /tmp/sphinx-err-syqhay9h.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

**Comment by AA-Turner** [comment]
Please test with [Sphinx 7.2.3](https://pypi.org/project/Sphinx/7.2.3/).

A

**Comment by Mab879** [comment]
The errors appears to be resolved with 7.2.3. Thanks for the quick fix!
